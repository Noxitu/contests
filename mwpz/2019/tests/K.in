3
2 12
100 200
1 1000000000