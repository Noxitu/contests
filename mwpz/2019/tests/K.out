27
102
609319791